src/components/step-nav.scss

```diff
@@ -0,0 +1,197 @@
+@import "../base/config.scss";
+
+//------------------------------------------------------------//
+
+// * StepNav Component
+// * 1. Settings
+// * 2. Define Default StepNav
+// * 3. Small Modifier
+// * 4. Title Modifier
+// * 5. Responsive Mixin
+
+// TODO : document global dependencies
+// TODO : Add Small StepNav Modifier
+// TODO : Add Titled Modifier
+
+//------------------------------------------------------------//
+
+// * 1. Settings
+
+$StepNav-size: 60px;
+$StepNav-border: 4px;
+
+$StepNav-size--small: 40px;
+$StepNav-border--small: 2px;
+
+// * 2. Define Default StepNav
+
+.StepNav {
+  padding-left: ($StepNav-size * 1.25);
+  position: relative;
+  @include clear();
+  padding-bottom: $vertical-block-spacing--small;
+  &.steps-2 {
+    .StepNav-Step {
+      width: 100%;
+    }
+  }
+  &.steps-3 {
+    .StepNav-Step {
+      width: 50%;
+    }
+  }
+  &.steps-4 {
+    .StepNav-Step {
+      width: 33%;
+    }
+  }
+}
+
+.StepNav-Step {
+  position: relative;
+  overflow: hidden;
+  list-style: none;
+  display: block;
+  float: left;
+  padding: 0;
+
+  &:before {
+    content: '';
+    width: 100%;
+    display: block;
+    position: absolute;
+    background: $ui-divider;
+    height: $StepNav-border;
+    top: ($StepNav-size / 2);
+    right: ($StepNav-size * 1.25);
+  }
+
+  &:first-child {
+    position: absolute;
+    left: 0;
+    top: 0;
+    .StepNav-Item {
+      float: left;
+    }
+    &:before {
+      display: none;
+    }
+  }
+
+  &.is-enabled {
+    .StepNav-Icon,
+    .StepNav-Title {
+      color: $color-primary;
+      transition: color .35s ease-in-out;
+    }
+    .StepNav-Icon:before {
+      border-color: $color-primary;
+      transition: border-color .35s ease-in-out;
+    }
+    &:before {
+      background-color: $color-primary;
+      transition: background-color .35s ease-in-out;
+    }
+  }
+
+  &.is-active,
+  &.is-enabled a:hover {
```

Comment:
not currently, it only becomes an `a` when enabled, so the future steps are just spans.